Makefile

```diff
@@ -22,36 +22,25 @@ export jupyter_port := 8888
 
 build-images: ## Build images
 
-	@ make build-image-debian
-	@ make build-image-fedora
+	@ make build-image
 
-build-image-debian: linux := debian
-build-image-fedora: linux := fedora
-
-build-image-debian build-image-fedora: _build-image
-
-_build-image:
+build-image:
 
 	docker build \
-		--file="$(project_directory)/docker/development/$(linux)/Dockerfile" \
-		--tag=$(docker_image_repository):$(docker_image_version)-$(linux) \
-		--tag=$(docker_image_repository):latest-$(linux) \
+		--file="$(project_directory)/docker/development/debian/Dockerfile" \
```

Comment:
Think it makes sense to remove the subfile altogether since we no longer need to disambiguate? (I know it's more of a pain, but might as well rip off the bandaid)

Reply:
Yeah, let's